src/docs/api/public.mdx

```diff
@@ -103,118 +109,230 @@ class OrganizationEventsEndpoint(...):
-      404: OpenApiResponse(description="A team with this slug already exists."),
-  },
-  examples=TeamExamples.CREATE_TEAM, # use examples defined in `src/sentry/apidocs/examples`
+    operation_id="Create a New Team",
+    parameters=[
+        GlobalParams.ORG_SLUG,
+        GlobalParams.name("The name for the team.", required=True),
+        GlobalParams.slug("Optional slug for the team."),
+    ],
+    request=TeamPostSerializer,
+    responses={
+        201: TeamSerializer,
+        400: RESPONSE_BAD_REQUEST,
+        403: RESPONSE_FORBIDDEN,
+        404: OpenApiResponse(description="A team with this slug already exists."),
+    },
+    examples=TeamExamples.CREATE_TEAM,
 )
-def post(self, request: Request, organization, member, team_slug: str,) -> Response:
+def post(self, request, organization, **kwargs):
     """
     Create a new team bound to an organization.
     """
 ```
 
-Specify an `operation_id` (this is what is shown as the title under the docs site sidebar).
+#### Title
+
+Specify an `operation_id` that will be shown as the title of the endpoint's page in the documentation.
+```python
+@extend_schema(
+    operation_id="Create a New Team",
+    ...
+)
+```
+
+#### Path and Query Parameters
 
-Specify the parameters using a list, and a serializer if needed.
+Specify the parameters using a list of parameters and serializers.
 
-- DRF serializers work very well for query parameters, try to use them where possible.
+- You can find existing parameters in this 
+[file](https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/parameters.py). Note that
+the `description` field in `OpenApiParameter` populates the parameter's description in the
+documentation.
+- DRF serializers conveniently translate into query parameters. See [here](https://github.com/getsentry/sentry/blob/0b9b6563bc4e232334cfc71f136a49117171d13f/src/sentry/api/endpoints/organization_stats_v2.py#L145)
+for an example of this.
+
+```python
+parameters=[
+    GlobalParams.ORG_SLUG,
+    GlobalParams.name("The name for the team.", required=True),
+    GlobalParams.slug("Optional slug for the team."),
+],
+```
 
-Specify the request and response serializers.
+#### Request Schema
 
-- Request serializers, since they use DRF serializers, can generally just be the serializer itself. There is plenty of customization and if needed can be overriden with `inline_serializer`. These will be passed to the `request` parameter in `extend_schema`.
-- For our response serializers, you must type the `serialize` function's return with a `TypedDict`
-  and pass the serializer as a parameter in `extend_schema`. See
-  [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/organization_member/scim.py)
+Specify the request serializer.
+
+- Request serializers, since they use DRF serializers, can generally just be the serializer itself.
+  There is plenty of customization and if needed can be overriden with `inline_serializer`. See
```

Comment:
Also `inline_serializer` could hyperlink to the actual docs for it here https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html#drf_spectacular.utils.inline_serializer